### DISCUSSION TITLE
How can I get the current cursor position

### DISCUSSION THREAD

**Discussion by Pixele9** [codesandbox/sandpack#440] [Q&A] [answered]
Is there a way to get the current cursor position and the file location? 
Say we have a file named: App.jsx and I'm in line 13 column 28
Can I look for the cursor's position to get something like this?: 

```
{
    "Ln": 13, 
    "Col": 28,
    "File": "App.jsx"
}
```
I want to use sockets to have in realtime the cursor's position moving, is there a way to achieve this?

**Comment by danilowoz** [comment, marked as answer]
Hey, I just updated the documentation with some instructions of how to do this:
https://sandpack.codesandbox.io/docs/advanced-usage/components#advanced-usage

Let me know if the documentation is clear enought

**Reply by Pixele9** [reply]
Thank you very much! I'll take a deeper look into it
